command/node_status.go

```diff
@@ -384,17 +392,26 @@ func (c *NodeStatusCommand) formatNode(client *api.Client, node *api.Node) int {
 
 }
 
+func (c *NodeStatusCommand) outputTruncatedNodeDriverInfo(node *api.Node) string {
+	drivers := make([]string, 1)
+
+	for driverName, driverInfo := range node.Drivers {
+		if !driverInfo.Healthy {
+			drivers = append(drivers, fmt.Sprintf("%s (unhealthy)", driverName))
+		} else {
+			drivers = append(drivers, driverName)
+		}
+	}
+	return strings.Trim(strings.Join(drivers, ", "), ", ")
```

Comment:
Why the trim. Join won't add one to the last element.

Reply:
Not sure I understand- Trim replaces all leading and trailing characters: https://golang.org/pkg/strings/#Trim